version/version.go

```diff
@@ -7,7 +7,7 @@ import (
 
 var (
 	// Version should be updated by hand at each release
-	Version = "0.7.0"
+	Version = "0.7.0.1"
```

Comment:
This should not be changed (in this PR, looks like we should update it otherwise though ;) )